### PR TITLE
Update 20_zigbee2mqtt-fails-to-start.md

### PR DESCRIPTION
**Pull request by Kosygor**
Rare case of having same /dev/serial/by-id device.

### CODE REVIEW

File: docs/guide/installation/20_zigbee2mqtt-fails-to-start.md [hunk 1]
```diff
@@ -186,3 +186,9 @@ The correct revision is: **E** like shown below.
 ![cc26xr1_revision](../../images/cc26xr1_revision.png)
 
 All earlier version are not supported (these are development boards). Return this board to the seller immediately.
+
+## Multiple cheap USB-UART
+
+If You have multiple devices connected that are running cheap USB-UART converters (CH341) they may be indistinguishable to your system, since they all possibly have the same idProduct, SerialNumber etc. thus they will share the same /dev/serial/by-id.
```
Comment (Koenkk) [inline docs/guide/installation/20_zigbee2mqtt-fails-to-start.md:192]: Do you have a solution for this issue?

**Comment by Kosygor** [comment]
Easiest solution is to change one of your devices to something with
different uart-usb converter. Second would be swapping converter or adding
external eeprom memory to chip that does not have one (ch341) thus you
would be able to add serial number.

sob., 30 gru 2023, 22:23 użytkownik Koen Kanters ***@***.***>
napisał:

**Comment by Koenkk** [comment]
@Kosygor please add that info to the FAQ

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**